### ISSUE TITLE
Error when running createSelfSigned.sh

### ISSUE THREAD

**Issue by Asuza** [dakshshah96/local-cert-generator#2]
Thanks for the guide and repo!

When running `sh createSelfSigned.sh`, I get the following error:
```
createSelfSigned.sh: 2: createSelfSigned.sh: Syntax error: "(" unexpected
```

After tweaking the script to remove the parens and redirect, it works, and looks like this:
```bash
#!/usr/bin/env bash
openssl req -new -sha256 -nodes -out server.csr -newkey rsa:2048 -keyout server.key -config server.csr.cnf
openssl x509 -req -in server.csr -CA rootCA.pem -CAkey rootCA.key -CAcreateserial -out server.crt -days 500 -sha256 -extfile v3.ext
```

My openssl version is 1.0.2g, running on Ubuntu Linux 17.10.

**Comment by kishoretulsiani** [comment]
I also got the same issue, workaround is to run first command with no -config option and enter all the required details on the screen

`
#!/usr/bin/env bash
openssl req -new -sha256 -nodes -out server.csr -newkey rsa:2048 -keyout server.key
openssl x509 -req -in server.csr -CA rootCA.pem -CAkey rootCA.key -CAcreateserial -out server.crt -days 500 -sha256 -extfile v3.ext
`

**Comment by adoussot** [comment]
it's because `sh` is an old shell, which doesn't seems to support *Process Substitution, <()* . You can do either:

- execute with a recent shell. e.g: `bash createSelfSigned.sh`, `zsh createSelfSigned.sh`
- make the script executable `chmod u+x createSelfSigned.sh` and execute it directly from your current modern shell
- don't use process substition and stick to `-config server.csr.cnf`

**Comment by dakshshah96** [comment]
@Asuza @kishoretulsiani This should be fixed in #3 😄 

Thanks @adoussot 🎉
